ui/src/index.css

```diff
@@ -11,3 +16,7 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.MuiCard-root {
```

Comment:
We discussed this before, were you not able to style  by using `classes` and `className` properties in the components themselves, instead of changing things globally? This may cause unwanted issues in other components.

What component are you trying to change? If you really need to use a CSS class, you can at least try to reduce the scope of the change by using something like this:
https://github.com/navidrome/navidrome/pull/953/files#diff-623dd661de804d0643ddaca55854069d421dc9b2f2fc04213c7f42204238d903R22-R33

Reply:
>  were you not able to style by using classes and className properties in the components themselves,

I'm sorry, but I wasn't able to go deeper than one level using the methods mentioned in the [docs]
(https://material-ui.com/customization/components/).
The only option I found was to override them using the `createMuiTheme`. But adding the style in pure CSS seemed way easier; 

> What component are you trying to change?

I was trying to fix this https://github.com/navidrome/navidrome/pull/941#issuecomment-812303419

> If you really need to use a CSS class, you can at least try to reduce the scope of the change by using something like this

Um, my thought was kind of in a different way. I felt like the default overflow hidden property in Cards could clash with the new overflow rules for future elements. So, in order to show the cards without any issues in the future, I made the style global. But now in retrospect, it seemed like a foolish thought. I'll add more specificity

Reply:
Fixed,